stdlib/Base64/src/encode.jl

```diff
@@ -69,10 +69,10 @@ function Base.unsafe_write(pipe::Base64EncodePipe, ptr::Ptr{UInt8}, n::UInt)::In
     i = 0
     p_end = ptr + n
     while true
-        buffer[i+1] = encode(b1 >> 2          )
```

Comment:
these are examples of unwanted alignment change

this can also happen at location of `=` instead of ()